CMakeLists.txt

```diff
@@ -448,6 +448,13 @@ message(STATUS "  Override with -DENABLE_INTERNAL_VOLK=ON/OFF")
 # Handle logging
 find_package(LOG4CPP REQUIRED)
 
+########################################################################
+# Disable complex math range checking
+########################################################################
+IF(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcx-limited-range")
+ENDIF()
+
```

Comment:
```cmake
check_cxx_compiler_flag("-fcx-limited-range" HAVE_CX_LIMITED_RANGE)                                                                          if(HAVE_CX_LIMITED_RANGE)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcx-limited-range")
endif()
```

is what we want, I think. What if clang adds the flag in some compiler version, we want CMake to figure that out for us.

Reply:
Also (I almost feel bad pointing that out), we have a CMake coding style too. CMake commands are lower-case, and our custom macros are uppercase.